spotify_player/src/media_control.rs

```diff
@@ -105,5 +114,116 @@ pub fn start_event_watcher(
     loop {
         update_control_metadata(&state, &mut controls, &mut track_info)?;
         std::thread::sleep(refresh_duration);
+
+        // this must be run repeatedly by your program to ensure
+        // the Windows event queue is processed by your application
+        #[cfg(target_os = "windows")]
+        windows::pump_event_queue();
+    }
+}
+
+// demonstrates how to make a minimal window to allow use of media keys on the command line
```

Comment:
Can you add note here mentioning that the `windows` handling codes in this file is copied from https://github.com/Sinono3/souvlaki/blob/master/examples/print_events.rs?

Reply:
Done